package.json

```diff
@@ -4,8 +4,8 @@
   "description": "Create a release",
   "main": "dist/index.js",
   "scripts": {
-    "lint": "eslint 'src/**.js' 'tests/**.js' --fix",
-    "test": "eslint 'src/**.js' 'tests/**.js' && jest --coverage",
+    "lint": "eslint src/**.js tests/**.js --fix",
+    "test": "eslint src/**.js tests/**.js && jest --coverage",
```

Comment:
The single quotes are necessary here to ensure the `**` are processed properly by eslint itself and not expanded by the shell. Versions of bash prior to 4 do not accept `**` and versions 4+ only handle them when `globstar` is set (it is not set by default).

Reply:
I didn't intend to commit this actually. I was having issues with the eslint version I had and that script (on windows). I can try it again.

Reply:
From what I can gather, single quotes are not cross platform (and fail on windows). I switched it to escaped double quotes which I believe _does_ work cross platform.